src/ashell/ide-comms.c

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2016-2017, Intel Corporation.
```

Comment:
As Geoff mentioned, this should just be 2017

Reply:
Done